test/write.jl

```diff
@@ -33,10 +33,10 @@ test_write(Dict(
 ))
 
 test_write(Dict(
-	"Complex128" => [1.0 -1.0 1.0+1.0im 1.0-1.0im -1.0+1.0im -1.0-1.0im 1.0im],
```

Comment:
Changing this seems OK but if the test fails without changing this then something is wrong.

Reply:
No that was just a name change to stay current with names and such. Tests pass with or without.